lib/fetching.js

```diff
@@ -1,4 +1,5 @@
 // Sets up the fetching module process and necessary event proxies.
+'use strict';
```

Comment:
Yes, we have `.eslintrc.json` as the style guide. I've been planning to gradually make style fixes as I touch various files. I don't want to rush and throw `'use strict'` directives everywhere at once since they can sometimes cause breaking changes.